src/includes/getting-started-verify/javascript.nextjs.mdx

```diff
@@ -0,0 +1,3 @@
+```javascript
+throw new Error("My first Sentry error!");
```

Comment:
This content is repeated in a handful of files with the exact same 3 lines. @PeloWriter do you have a suggestion on how to reuse this as an include or something else?

Reply:
For the guides that fall back to JavaScript, the verification code sample can be removed if it's identical to the one provided for JS (I am removing the snippet for Cordova, for example, since it's entirely a duplicate).